staging/src/k8s.io/kubectl/pkg/describe/describe.go

```diff
@@ -4143,21 +4221,17 @@ func describeDeployment(d *appsv1.Deployment, selector labels.Selector, internal
 			ru := d.Spec.Strategy.RollingUpdate
 			w.Write(LEVEL_0, "RollingUpdateStrategy:\t%s max unavailable, %s max surge\n", ru.MaxUnavailable.String(), ru.MaxSurge.String())
 		}
-		DescribePodTemplate(&internalDeployment.Spec.Template, w)
+		DescribePodTemplate(&d.Spec.Template, w)
```

Comment:
`internalDeployment` was just a deep copy of `d`, and it is only used here. Since this is just a printer and does not mutate, I don't understand why we'd need to deep copy. If there's a reason, I could add it back right here.